evap/evaluation/models.py

```diff
@@ -2120,24 +2181,25 @@ def construct_mail(self, to_email, cc_addresses, subject_params, body_params):
         )
 
     @classmethod
-    def send_reminder_to_user(cls, user, first_due_in_days, due_evaluations):
+    def send_reminder_to_user(
+        cls, user: UserProfile, first_due_in_days: int, due_evaluations: Iterable[Evaluation]
+    ) -> None:
         template = cls.objects.get(name=cls.STUDENT_REMINDER)
         subject_params = {"user": user, "first_due_in_days": first_due_in_days}
         body_params = {"user": user, "first_due_in_days": first_due_in_days, "due_evaluations": due_evaluations}
```

Comment:
Should we use `Collection[Evaluation]` for `due_evaluations` here too then? I suppose the email template will iterate at least twice, once for each language

Reply:
We should check all our uses of `Iterable` anyways. I wonder if Python typing could ever have something like "calls `__iter__` at most once"

Reply:
`Collection` has the problem that `QuerySet`s are not collections, so typeguard will fail for them in tests. I don't know, I don't want to block the feature for this. Will discuss separately (and maybe make an issue)